Doc/library/sqlite3.rst

```diff
@@ -304,8 +304,9 @@ Module functions
    :param bool check_same_thread:
        If ``True`` (default), only the creating thread may use the connection.
        If ``False``, the connection may be shared across multiple threads;
-       if so, write operations should be serialized by the user to avoid data
-       corruption.
+       if so, write operations may need to be serialized by the user,
```

Comment:
What about thinking outsize the "ized" and stating more explicitly what the user needs to do, instead of how they should do it?

```suggestion
       if so, write operations must only be performed in series,
```

or

```suggestion
       if so, write operations must only be performed serially,
```

or

```suggestion
       if so, the user should ensure write operations are only performed serially,
```


Or what about instead flipping it and talking about what the user needs to ensure _doesn't_ happen?

```suggestion
       if so, write operations must not performed in parallel,
```

or 

```suggestion
       if so, the user should ensure write operations are not performed in parallel,
```

Reply:
Marking as resolved (outdated)